setup.py

```diff
@@ -0,0 +1,15 @@
+from setuptools import setup, find_packages
+
+# Read the contents of requirements.txt
+with open("requirements.txt") as f:
+    requirements = f.read().splitlines()
+
+setup(
+    name="dl4ds_tutor",
+    version="0.1.0",
+    packages=find_packages(),
+    package_dir={"modules": "modules"},
```

Comment:
where does the actual package name come from? This line? Since it shows up in pip as "modules". If so, change to "edubot-core".

Reply:
Sure. we can do that after we transfer the repo.